app/controllers/employees_controller.rb

```diff
@@ -6,6 +6,12 @@ def new
   def create
     @employee = Employee.new(employee_params)
 
+    unless EmployeeFinder.new.employee_exists?(@employee.slack_username)
```

Comment:
would it be possible to change this to an `if` instead of an `unless`? I find `unless` statements hard to parse because there's a negative - easier to say "if this, then that" rather than "unless not this, then that" (as I write this, I am not sure if that is even correct...more evidence that unless is hard!)

Also, since we are using the same statement on this line and L35, perhaps the logic can be extracted into a private method with a nice name....